Add timestampFlag docs (#997)
urfave/cli: docs/v2/manual.md

is checked by the cli internals in order to print the `App.Version` via
`cli.VersionPrinter` and break execution.

### Timestamp Flag

Using the timestamp flag is simple, You can look at time.Parse to get layout examples : https://golang.org/pkg/time/#example_Parse 

<!-- {

>   "args": ["&#45;&#45;meeting", "2019-08-12T15:04:05"],

Reviewer: ⭐⭐⭐ _(must change)_ It took me a few seconds to figure out what I was reading here, can you add an example invoke to these docs? Like this:

```
myapp --meeting "2019-08-12T15:04:05"
```

}
```

### Timestamp Flag

> Using the timestamp flag is simple, You can look at time.Parse to get layout examples : https://golang.org/pkg/time/#example_Parse 

Reviewer 2: ```suggestion
Using the timestamp flag is simple. Please refer to [`time.Parse`](https://golang.org/pkg/time/#example_Parse) to get possible formats.
```

<!-- {
  "args": ["&#45;&#45;meeting", "2019-08-12T15:04:05"],
  "output": "2019\\-08\\-12 15\\:04\\:05 \\+0000 UTC"
} -->
``` go
package main

import (
  "fmt"
  "log"
  "os"

  "github.com/urfave/cli/v2"
)

func main() {
  app := &cli.App{
    Flags: []cli.Flag {
      &cli.TimestampFlag{Name: "meeting", Layout: "2006-01-02T15:04:05"},
    },
    Action: func(c *cli.Context) error {
      fmt.Printf("%s", c.Timestamp("meeting").String())
      return nil
    },
  }

  err := app.Run(os.Args)
  if err != nil {
    log.Fatal(err)
  }
}
```

> In this example the flag could be used like so : 

Reviewer 2: ```suggestion
In this example the flag could be used like this:
```

`myapp --meeting 2019-08-12T15:04:05`

> side note : quotes may be necessary around the date depending on your layout (if you have spaces for instance)

Reviewer 2: ```suggestion
Side note: quotes may be necessary around the date depending on your layout (if you have spaces for instance)
```
